README.md

```diff
@@ -11,16 +11,20 @@ c.GetApps() // returns a map[String]fargo.Application
 # Testing
 
 Tests can be executed using docker container. See the below section to build and start 
-all the required containers. Once fargo container is running:
+all the required containers. Once eureka containers are running, and fargo container is built.
```

Comment:
- [ ] Capitalize "Eureka" as a proper noun.
- [ ] s/Once Eureka/Once the Eureka/
- [ ] s/container is built./image is built,/  
  For the last one, did you mean to say that once the containers are running and the _fargo_ image is built, _then_ you can run the command that follows?